packages/webview_flutter/webview_flutter_platform_interface/lib/v4/src/platform_navigation_callback_delegate.dart

```diff
@@ -0,0 +1,91 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/foundation.dart';
+import 'package:plugin_platform_interface/plugin_platform_interface.dart';
+
+import 'webview_platform.dart';
+
+/// An interface defining navigation events that occur on the native platform.
+///
+/// The [WebViewControllerDelegate] is notifying this delegate on events that
+/// happened on the platform's webview. Platform implementations should
+/// implement this class and pass an instance to the [WebViewControllerDelegate].
+abstract class PlatformNavigationCallbackDelegate extends PlatformInterface {
```

Comment:
This should be `PlatformNavigationDelegate`, right?

Reply:
Not sure, the original name was `NavigationCallbackHandlerDelegate`. Here we concluded that the post-fixes `Handler` and `Delegate`  synonymous and therefore we should remove either one of them. 

We decided to stick with `Delegate` and prefix the word `Platform`. The question is are we delegating navigation or  callbacks as a result of navigation. If the first case then `PlatformNavigationDelegate` makes sense, otherwise I think `PlatformNavigationCallbackDelegate` is more appropriate.

@stuartmorgan do you mind chiming in?

Reply:
It's functionally a navigation delegate (e.g., it's how a client implements `onNavigationRequest`); the details of whether that delegation is done via inheritance or settable callbacks seems unimportant from a naming standpoint IMO. So I would call it `PlatformNavigationDelegate`.

Reply:
Thanks, I update the code accordingly in commit 3e22600 (figured I could revert it if necessary).